examples/recruiting/views.py

```diff
@@ -75,7 +75,7 @@ def is_available_within_months(  # pylint: disable=W0602, C0116, W9011
         end = start + relativedelta(months=months)
         return Candidate.available_from.between(start, end)
 
-    def list_few_shots(self) -> List[FewShotExample]:  # pylint: disable=W9011
+    def list_few_shots(self) -> List[FewShotExample]:  # pylint: disable=W9011, C0116
```

Comment:
I think the docstring for this method is going to be short enough to put it here instead of disabling the linter.